examples/00-core_examples/00-basic_dpf_implementation.py

```diff
@@ -0,0 +1,217 @@
+# # ~~~~~~~ DPF PRESENTATION ~~~~~~~ # #
+
+# Ansys Data Processing Framework (DPF) provides a modular and easy to use toolbox for accessing and transforming
+# simulation data using the DPF operators.
+# DPF is a workflow-based framework which allows simple and/or complex evaluations by chaining operators.
+
+# DPF is developed around two core components:
+# - Data represented as a *field* (Component based on vectors quantities where the data is defined)
+# - An *operator* to act upon this data
+# Operators are like verbs, acting on the data, while fields are like nouns, objects that hold data.
+
+######################################################################################################################
+# # ~~~~~~~ DPF CODE ORGANISATION ~~~~~~~ # #
+
+# DPF package is named : 'ansys.dpf.core package'. This package contains (based in the 2024 doc version):
+# a) Three subpackages:
+#       - 'ansys.dpf.core.examples' package
+#       - 'ansys.dpf.core.helpers' namespace
+#       - 'ansys.dpf.core.operators' package
+# b) Sixty three submodules
+#       - 'animation', 'animator', 'any', 'available_result', 'collection', 'collection_base', 'common',
+#       'Operator config', 'core', 'custom_fields_container', 'custom_operator_base', 'custom_type_field',
+#       'cyclic_support', 'data_sources', 'data_tree', 'dimensionality', 'dpf_array', 'dpf_operator', 'elements',
+#       'element_descriptor', 'errors', 'faces', 'field','fields_definition', 'fields_container',
+#       'fields_container_factory', 'fields_factory', 'generic_data_container', 'generic_support','normalize_vector',
+#       'create_point', 'incremental', 'inputs', 'meshed_region', 'meshes_container', 'mesh_info',
+#       'mesh_scoping_factory', 'model', 'nodes', 'operator_specification', 'outputs', 'path_utilities', 'plotter',
+#       'plugins', 'property_field', 'results', 'result_info', 'runtime_config', 'scoping', 'scopings_container',
+#       'server', 'server_context', 'server_factory', 'server_types', 'session', 'settings', 'streams_container',
+#       'string_field', 'support', 'time_freq_scoping_factory', 'time_freq_support', 'unit_system', 'workflow'
```

Comment:
not sure this should be in the example. 
We can describe this in the doc, like in the user guide.
The `ansys.dpf.core.operators` submodule can be presented in the example on operators, to explain that their Python wrappers are there.
The `ansys.dpf.core.helpers` I think we can make several dedicated examples about.
And the `ansys.dpf.core.examples` submodule may require a short introduction in the main page for examples, but not much more.